simulation_parameters/Constants.cs

```diff
@@ -84,7 +84,17 @@ public static class Constants
     ///   Extra radius added to the spawn radius of things to allow them to move in the "wrong" direction a bit
     ///   without causing them to despawn instantly
     /// </summary>
-    public const int DESPAWN_RADIUS_OFFSET_SQUARED = 2500;
+    public const int DESPAWN_RADIUS_OFFSET = 50;
+
+    /// <summary>
+    ///   Squared radius. Microbes despawn if they are outside it.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     The value should be (MICROBE_SPAWN_RADIUS + DESPAWN_RADIUS_OFFSET)^2
```

Comment:
Why not make this math then be that? As long as those variables are also const, you can put the calculation in here so that it cannot get out of sync.

Reply:
Right, that makes more sense. I'll do that. I think I can remove the summary then because the formula and names explain the constant.

Reply:
"Microbes despawn if they are outside it" is pretty useful comment, so putting that on the despawn radius if it is missing from there would be nice.